core/storage/base_model/gae_models.py

```diff
@@ -131,13 +131,14 @@ class BaseModel(datastore_services.Model):
     # Whether the current version of the model instance is deleted.
     deleted = datastore_services.BooleanProperty(indexed=True, default=False)
 
-    def __init__(self, *args, **kwargs):
-        # type: (*Any, **Any) -> None
+    # We use type Any for *args and **kwargs to denote compatibility with the
+    # overidden constructor of the parent class i.e datastore_services.Model
```

Comment:
Spelling: overridden

Reply:
Sorry, fixed this and 2 other places now. There were no more instances in the PR.